src/ceph.in

```diff
@@ -172,24 +173,24 @@ def raw_write(buf):
     sys.stdout.buffer.write(buf)
 
 
-def osdids():
-    ret, outbuf, outs = json_command(cluster_handle, prefix='osd ls')
+def osdids(h):
+    ret, outbuf, outs = json_command(h, prefix='osd ls')
     if ret:
         raise RuntimeError('Can\'t contact mon for osd list')
     return [line.decode('utf-8') for line in outbuf.split(b'\n') if line]
 
 
-def monids():
-    ret, outbuf, outs = json_command(cluster_handle, prefix='mon dump',
+def monids(h):
```

Comment:
this breaks the test of `cephtool/test.sh`, like
```
2022-07-27T09:23:37.246 INFO:tasks.workunit.client.0.smithi154.stderr:/home/ubuntu/cephtest/clone.client.0/qa/workunits/cephtool/test.sh:35: expect_false:  set -x
2022-07-27T09:23:37.247 INFO:tasks.workunit.client.0.smithi154.stderr:/home/ubuntu/cephtest/clone.client.0/qa/workunits/cephtool/test.sh:36: expect_false:  ceph ping mon.foo
2022-07-27T09:23:37.312 INFO:tasks.workunit.client.0.smithi154.stderr:[errno 2] RADOS object not found (error calling ping_monitor)
2022-07-27T09:23:37.313 INFO:tasks.workunit.client.0.smithi154.stderr:/home/ubuntu/cephtest/clone.client.0/qa/workunits/cephtool/test.sh:36: expect_false:  return 0
2022-07-27T09:23:37.313 INFO:tasks.workunit.client.0.smithi154.stderr:/home/ubuntu/cephtest/clone.client.0/qa/workunits/cephtool/test.sh:2639: test_mon_ping:  ceph ping 'mon.*'
2022-07-27T09:23:37.415 INFO:tasks.workunit.client.0.smithi154.stderr:monids() missing 1 required positional argument: 'h'
```

see https://pulpito.ceph.com/kchai-2022-07-27_06:59:49-rados-wip-kefu-testing-2022-07-27-0857-distro-default-smithi/